pipeline/init.bats

```diff
@@ -92,23 +92,22 @@ teardown() {
             CK8S_CONFIG_PATH="${config_2}" run ck8s init
             assert_success
 
-            run diff "${config_1}/defaults/common-config.yaml" "${config_2}/defaults/common-config.yaml"
+            run diff <(sort "${config_1}/defaults/common-config.yaml") <(sort "${config_2}/defaults/common-config.yaml")
```

Comment:
Is the sort really needed, shouldn't we strive for it to not introduce unnecessary changes instead?

Reply:
Yes that would be preferred, but Im unsure how to make yq4 not change order on things.

Reply:
Okay, great. Thought too try it out now. And no have not tested it with the new changes. Before when i did a reinit it moved some stuff. Will update the PR once i have tested it.

Reply:
It worked. Wonder what fixed it :sweat_smile: